Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocBooleanValueImplementation/Rendering/BocBooleanValueRendererTest.cs

```diff
@@ -435,7 +436,7 @@ private void CheckLinkAttributes (XmlNode link, string checkedState, string desc
     private void AssertValidationErrors (XmlNode node)
     {
       Html.AssertAttribute (node, StubValidationErrorRenderer.ValidationErrorsIDAttribute, c_clientID + "_ValidationErrors");
-      Html.AssertAttribute (node, StubValidationErrorRenderer.ValidationErrorsAttribute, c_validationErrors);
+      Html.AssertAttribute (node, StubValidationErrorRenderer.ValidationErrorsAttribute, s_validationErrors.GetValue());
```

Comment:
The fake-implementation renderes attributes for the validation errors. I think we should align test and fake somehow because right now it works mainly because the string doesn't contain to-be-escaped chars. Would ToString() be more correct?